src/index.js

```diff
@@ -64,23 +62,19 @@ const addTask = () => {
   displayTasks(list);
   input.value = '';
 };
-
 addBtn.addEventListener('click', addTask);
 form.addEventListener('submit', (e) => {
   e.preventDefault();
   addTask();
   input.blur();
 });
-
 // remove function
 const deleteTask = (idNumer, e) => {
   removeTask(list, idNumer - 1);
-
   // Update Index
   list.forEach((task, i) => {
     task.index = i + 1;
   });
-
   // Update id
   const binBtns = document.querySelectorAll('.more_btn');
   binBtns.forEach((btn) => {
```

Comment:
- [ optional] if you don't mind, you can actually use a trash-can icon here for the remove button so as to make it user-friendly instead of the `:` great job. keep it up 👏🏾